src/bindings/js/node/scripts/download_runtime.js

```diff
@@ -281,4 +281,4 @@ function unarchive(tarFilePath, dest) {
   });
 }
 
-module.exports = { downloadRuntime };
+module.exports = { downloadRuntime, downloadFile, checkIfDirectoryExists };
```

Comment:
You need to rename function `checkIfDirectoryExists ` if you check not directory existence